### PR TITLE
Add linter for kotlin

### PR DESCRIPTION
**Pull request by ripzery**
Issue/Task Number: `159-add-lint-for-kotlin`

# Overview

Add linter for Kotlin

# Changes

- Add plugin [ktlint-gradle](https://github.com/jlleitschuh/ktlint-gradle)
- Fix lint error

# Implementation Details

`N/A`

# Usage

Simply run `./gradlew ktlintCheck` under project root directory.

# Impact

`N/A`

### CODE REVIEW

File: omisego-sdk/src/test/kotlin/co/omisego/omisego/OMGAPIClientTest.kt [hunk 1]
```diff
@@ -145,7 +147,7 @@ class OMGAPIClientTest {
     @Test
     fun `OMGAPIClient should be executed when API return success true correctly`() {
         val expected = gson.fromJson<OMGResponse<User>>(userFile.readText(),
-                object : TypeToken<OMGResponse<User>>() {}.type)
+            object : TypeToken<OMGResponse<User>>() {}.type)
```
Comment (badoualy) [inline omisego-sdk/src/test/kotlin/co/omisego/omisego/OMGAPIClientTest.kt]: What do you think of
```kotlin
val expected = gson.fromJson<OMGResponse<User>>(
    userFile.readText(),
    object : TypeToken<OMGResponse<User>>() {}.type
)
```

This way, arugments would be aligned, easier to read 🤔

File: omisego-sdk/src/test/kotlin/co/omisego/omisego/OMGAPIClientTest.kt [hunk 2]
```diff
@@ -157,7 +159,7 @@ class OMGAPIClientTest {
     fun `OMGAPIClient should be executed when API return success false correctly`() {
         expectedEx.expect(OMGAPIErrorException::class.java)
         val apiError = APIError(ErrorCode.CLIENT_INVALID_AUTH_SCHEME,
-                "The provided authentication scheme is not supported")
+            "The provided authentication scheme is not supported")
```
Comment (badoualy) [inline omisego-sdk/src/test/kotlin/co/omisego/omisego/OMGAPIClientTest.kt]: Same here. Weirdly (intentionally?) Intellij/Android Studio doesn't allow the option to align multiline arguments like you can do in java, giving cases like this, where parameters are not aligned 😢

File: omisego-sdk/src/test/kotlin/co/omisego/omisego/OMGAPIClientTest.kt (hunk 1 above)
Comment (ripzery) [reply omisego-sdk/src/test/kotlin/co/omisego/omisego/OMGAPIClientTest.kt]: Fixed

File: omisego-sdk/src/test/kotlin/co/omisego/omisego/OMGAPIClientTest.kt (hunk 2 above)
Comment (ripzery) [reply omisego-sdk/src/test/kotlin/co/omisego/omisego/OMGAPIClientTest.kt]: Yeah it doesn't give me a warning, I need to keep it in mind 😢

**Comment by ripzery** [comment]
Android Studio 3.1 has just released and it comes with official Kotlin lint 😒

**Pull request merged**